upgrader/upgrade-061-062/scripts/upgradeKeptn.sh

```diff
@@ -0,0 +1,131 @@
+  "https://raw.githubusercontent.com/keptn/keptn/release-$KEPTN_VERSION/installer/manifests/keptn/keptn-ingress.yaml"
+  "https://raw.githubusercontent.com/keptn/keptn/release-$KEPTN_VERSION/installer/manifests/keptn/uniform-services-openshift.yaml"
+  "https://raw.githubusercontent.com/keptn/keptn/release-$KEPTN_VERSION/installer/manifests/keptn/uniform-distributors-openshift.yaml"
+  )
+
+for manifest in "${manifests[@]}"
+do
+   :
+   if curl --head --silent -k --fail $manifest 2> /dev/null;
+     then
+      continue
+     else
+      print_error "Required manifest $manifest not available. Aborting upgrade."
+      exit 1
+    fi
+done
+
+
+print_debug "Check if Keptn 0.6.1 is currently installed"
+API_IMAGE=$(kubectl get deployment -n keptn api -o=jsonpath='{$.spec.template.spec.containers[:1].image}')
+
+  if [[ $API_IMAGE != 'keptn/api:0.6.1' ]]; then
+    print_error "Installed Keptn version does not match 0.6.1. aborting."
+    exit 1
+  fi
+
+
+print_debug "Updating Keptn core."
+kubectl apply -f https://raw.githubusercontent.com/keptn/keptn/release-$KEPTN_VERSION/installer/manifests/keptn/core.yaml
+kubectl apply -f https://raw.githubusercontent.com/keptn/keptn/release-$KEPTN_VERSION/installer/manifests/keptn/quality-gates.yaml
+
+kubectl -n keptn delete deployment api
+kubectl -n keptn delete service api
+
+kubectl get namespace openshift
+  if [[ $? == '0' ]]; then
+    print_debug "OpenShift platform detected. Updating OpenShift core services"
+    kubectl apply -f https://raw.githubusercontent.com/keptn/keptn/release-$KEPTN_VERSION/installer/manifests/keptn/uniform-services-openshift.yaml
+    kubectl apply -f https://raw.githubusercontent.com/keptn/keptn/release-$KEPTN_VERSION/installer/manifests/keptn/uniform-distributors-openshift.yaml
+  fi
+
+DOMAIN=$(kubectl get configmap -n keptn keptn-domain -ojsonpath="{.data.app_domain}")
+
+# check if full installation is available
+kubectl -n keptn get svc gatekeeper-service
+
+  if [[ $? == '0' ]]; then
+      print_debug "Full installation detected. Upgrading CD and CO services"
+      kubectl apply -f https://raw.githubusercontent.com/keptn/keptn/release-$KEPTN_VERSION/installer/manifests/keptn/continuous-deployment.yaml
+      kubectl apply -f https://raw.githubusercontent.com/keptn/keptn/release-$KEPTN_VERSION/installer/manifests/keptn/continuous-operations.yaml
+      curl -k https://raw.githubusercontent.com/keptn/keptn/release-$KEPTN_VERSION/installer/manifests/keptn/keptn-api-virtualservice.yaml | \
+        sed 's~DOMAIN_PLACEHOLDER~'"$DOMAIN"'~' | kubectl apply -f -
+  else
+      print_debug "Quality gates installation detected. Upgrading Nginx ingress"
+      kubectl apply -f https://raw.githubusercontent.com/keptn/keptn/release-$KEPTN_VERSION/installer/manifests/keptn/keptn-ingress.yaml
+
+      curl -k https://raw.githubusercontent.com/keptn/keptn/release-$KEPTN_VERSION/installer/manifests/keptn/keptn-ingress.yaml | \
```

Comment:
Here we have to be careful because $DOMAIN can contain a port. Correct me if I am wrong.
But if so, the domain.placeholder must not contain a port.